docs/improving-gci-experience.md

```diff
@@ -0,0 +1,377 @@
+
+### Is GCI all about the number of tasks?
+
+In the [GCI
+rules](https://developers.google.com/open-source/gci/resources/contest-rules)
+there are two important factors for choosing finalists and grand-prize
+winners.
+
+* Number of tasks
+  > Each complete task entry is worth one (1) point.
+
+  In order to be considered for the grand-prize, a student needs to be in
+  the top-10 in terms of number of tasks.
+
+* Other factors
+  > Each entry will be judged according to the following factors: creativity,
+  thoroughness, and quality of work. The Organization may additionally factor
+  the contestant’s involvement in the Organization’s community in the judging.  
+
+While the number of tasks that they complete is important, the winning
+students are often chosen for not only the number of tasks that they
+completed but also their willingness to help other students and participate
+in the community. Thus being in the top-10 is a requirement, but the
+student with the most completed tasks won't necessarily win the grand-prize.
+
+## How do I make sure none of my contributions are forgotten?
+
+If you are worried that some of your contributions may not be taken into
+account when choosing the winners, you can create a document which lists
+the achievements that were not GCI tasks.
+
+You can see an example format for this document
+[here](https://paper.dropbox.com/doc/GCI-achievements-example-CT1htUQQnkFkZE1k27ql2).
+If you create your achievement document, you can add it to [this
+list](https://docs.google.com/spreadsheets/d/1eF2xi5_ZrpURH5N_VV99C8o-deLh-A5YjNwWLc4opik/edit?usp=sharing).
+
+**Note**: This is not a requirement for the GCI students. We simply want to
+give students a simple way of showcasing all of their achievements. We will
+still review Zulip pull requests and issues, and Zulip threads in addition
+to the GCI tasks they've completed.
+
+## Why should I claim a task before working on it?
+
+The [GCI
+rules](https://developers.google.com/open-source/gci/resources/contest-rules)
+require you to claim a task before you start working on it. It's recommended
+since students can't see the number of instances available and it may happen
+that someone else claims the last instance before you can claim it. Your work
+won't then be applicable for review.
+
+The workflow for completing GCI tasks:
+
+* Student claims the task.
+
+* Student works on the tasks and makes sure it fulfills all the requirements.
+
+* Before submitting the task, they check that the task passes all the tests
+(including Travis tests and manual testing of the feature).
+
+* Student submits the task in the GCI app and waits for a review - they
+should focus on other endeavors while waiting.
+
+* The task is going through the review process, with refactoring and
+improvements required by mentors.
+
+* The task is accepted when a mentor assesses it as fulfilling the task
+requirements - this doesn't necessarily mean the pull request will be merged
+at the same time.
```

Comment:
Might add a bit of explanation of why here, explaining that sometimes the best expert to do a final review and merge the PR isn't available, but the mentors who are available are reasonably confident the work is complete.  

Also would add that we still expect students to address any feedback after the task is approved.